canonical-web-and-design/ubuntu.com
# Fix the blog feeds

## Done
Added redirect for the blog feed urls
Updated the old feed redirects

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Go to /blog/feed and see it downloads
- Go to /blog/topics/design/feed and check that redirects
